src/styles/_catalog.scss

```diff
@@ -1,22 +1,33 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
   margin: 0 auto;
+  max-width: 944px;
+  gap: 48px;
+
+  @media (max-width: 488px)  {
+    grid-template-columns: $card-width;
+  }
+
+  @media (min-width: 488px)  {
+    grid-template-columns: repeat(2, $card-width)
+  }
+
+  @media (min-width: 768px)  {
+    grid-template-columns: repeat(3, $card-width)
+  }
+
+  @media (min-width: 1024px)  {
+    grid-template-columns: repeat(4, $card-width)
+  }
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
-    height: 300px;
+    display: grid;
+    place-items: center;
+    @include card-sizes;
```

Comment:
You don`t need to use width here

Reply:
I put height in advanced if we gonna to change dimenstions for cards